cfme/services/requests.py

```diff
@@ -217,28 +194,40 @@ def debug_requests():
             logger.debug(' %s', row)
 
 
-def find_request(cells):
-    """Finds the request and opens the page
-
-    See :py:func:`wait_for_request` for futher details.
+def find_request(cells, partial_check=False):
+    """Finds the request and returns the row element
 
     Args:
         cells: Search data for the requests table.
-    Returns: Success of the action.
+        partial_check: If to use the ``in`` operator rather than ``==`` in find_rows_by_cells().
+    Returns: row
     """
     navigate_to(Request, 'All')
     for page in paginator.pages():
-        try:
-            # found the row!
-            row, = fields.request_list.find_rows_by_cells(cells)
-            return True
-        except ValueError:
+        # We check only for the SplitTable. Can't think of better detection.
+        if version.current_version() < "5.5.0.8"\
```

Comment:
**REQUIRED:** As we are reworking this, let's remove the 5.5 logic as 5.5 is no longer supported